README.md

```diff
@@ -1,189 +1,249 @@
-# c2paTool
+# c2patool - C2PA command line tool
 
-Command line tool for displaying and adding C2PA manifests  
-A file path to a JPEG or a claim definition JSON file must be provided  
-If a JPEG path is given, this will generate a summary report of any manifests in that file  
-If a manifest definition JSON file is specified, the manifest will be created and displayed in a JSON report
+c2patool is a command line tool for working with C2PA [manifests](https://c2pa.org/specifications/specifications/1.0/specs/C2PA_Specification.html#_manifests). Currently, the tool supports:
 
-## Displaying Manifest data
+- Reading a JSON report of C2PA manifests in an JPEG or PNG image
+- Reading a low-level report of C2PA manifest data in a JPEG or PNG image
+- Previewing manifest data from a JSON config file
+- Adding a C2PA manifest to an existing JPEG or PNG image
+- Writing a C2PA manifest to a `.c2pa` manifest file
 
-Invoking the tool with a path to an image file will output a JSON report of the Manifests in the file
-File formats supported are jpeg and png. 
+## Installation
 
-```c2patool image.jpg```
+### via Homebrew
 
-## Displaying detailed Manifest data
+You can install c2patool on Mac and Linux via [Homebrew](https://brew.sh/):
 
-The -d option will output a detailed JSON report of the internal C2PA structure
+```shell
+brew tap contentauth/tools
+brew install c2patool
+```
 
-```c2patool image.jpg  -d```
+### Building from source
```

Comment:
Technically the  homebrew install is also a build from source that will install the whole rust build system as well.

Reply:
I'm talking with @scouten-adobe on Thursday to see if we can make it download the binaries from the build action so we can bypass builds for the most common platforms.